presto-main/src/test/java/com/facebook/presto/operator/TestHashJoinOperator.java

```diff
@@ -1527,7 +1527,7 @@ public boolean filter(int leftPosition, Page leftPage, int rightPosition, Page r
         }
     }
 
-    private static class DummySpillerFactory
+    static class DummySpillerFactory
```

Comment:
ditto about refactorings

Reply:
I need this for TestHashBuilderOperator

Reply:
@arhimondr is this resolved since it's necessary for the test to compile?

Reply:
yes